### DISCUSSION TITLE
Request: Be able to "hide" line items

### DISCUSSION THREAD

**Discussion by GOTROK** [File-New-Project/EarTrumpet#643] [Ideas]
Would be nice to be able to set the volume of a given app and then hide it so it doesn't show on the main app screen.  This would allow you to only show the apps that might need their volume adjusted.

Example:  I use skype for chat, but not for conferences/video/calls.  Would be good to set that to a low volume and then hide it so it doesn't clutter the app.  Conversely, I use WebEx for conferences so would need it visible to adjust sound as needed during a call.

**Comment by 2-X** [comment]
I would also like this, but for a different reason. I have a few programs that I know I don't ever want to adjust the volume for, so I want to hide them from showing up in EarTrumpet.
